kuma/wiki/views/misc.py

```diff
@@ -84,7 +84,7 @@ def autosuggest_documents(request):
 
     # Generates a list of acceptable docs
     docs_list = []
-    for doc in docs:
+    for doc in docs[:100]:
```

Comment:
I would say we could make this even smaller, say somewhere between `25` and `50`.

Reply:
You're not wrong but it's not important enough at the moment. :) 
Thanks!